HACKING.md

```diff
@@ -4,70 +4,84 @@ Hacking on `snapd` is fun and straightforward. The code is extensively unit
 tested and we use the [spread](https://github.com/snapcore/spread)
 integration test framework for the integration/system level tests.
 
-## Development
+## Setting up
 
 ### Supported Ubuntu distributions
 
 Ubuntu 18.04 LTS or later is recommended for `snapd` development.
+Usually, the latest LTS would be the best choice.
 
-If you want to build or test on older versions of Ubuntu, additional steps
+>If you want to build or test on older versions of Ubuntu, additional steps
 may be required when installing dependencies.
 
 
 ### Supported Go version
 
-Go 1.13 or later is required to build `snapd`.
+[Go 1.13](https://pkg.go.dev/go@go1.13) or later is required to build `snapd`.
```

Comment:
Regarding the link issue, I am not sure I understand. This link brings you to the documentation of this version on the `pkg.go.dev`. The idea was to spare the reader (should they attempt searching for what Go 1.13 really is), since this link will bring them to the exact place. I thought it's useful especially since I mention another version below, and Go 1.13 is not the latest version of Go.

Regarding _the_ thing, people with Russian origins are known to be horrible with all this "article" thing :-). But jokes aside, I was told by one technical writer that in titles and presentation bullets it is acceptable to omit article of any sort, since it makes the statement nominal. But I can be wrong (@degville could you comment?). In any way, I don't see any problem with adding _the_. :-)